desktop/src/main.rs

```diff
@@ -124,8 +138,13 @@ fn run() -> Result<(), Error> {
 fn init_debugger(args: &Args) -> Result<Option<TuiDebugger>, Error> {
     // Initialize global logger. The logger kind depends on the `--debug` flag.
     debug::init_logger(args.debug);
+
+    #[cfg(debug_assertions)]
     log::set_max_level(log::LevelFilter::Trace);
 
+    #[cfg(not(debug_assertions))]
+    log::set_max_level(log::LevelFilter::Error);
```

Comment:
I'd say `Warn` is a better level for release mode.

Reply:
I think that's a matter of taste. I try to worry the user as little as possible. And warnings can make the user feel uncomfortable using the emulator (which is not necessary for things we pump out warnings at the moment (e.g. writing to the cartridge)).

Reply:
Ah, good to know!